SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorPackageAssembly.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Logging/LogMacros.h"
+#include "Misc/MonitoredProcess.h"
+#include "Widgets/Notifications/SNotificationList.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKEditorPackageAssembly, Log, All);
+
+class SPATIALGDKEDITOR_API FSpatialGDKPackageAssembly : public TSharedFromThis<FSpatialGDKPackageAssembly>
+{
+public:
+	FSpatialGDKPackageAssembly();
+
+	bool CanBuild() const;
+
+	void BuildAllAndUpload(const FString& AssemblyName, const FString& Configuration, const FString& AdditionalArgs, bool bForce);
```

Comment:
Could we rename bForce bForceOverwrite. I only understood what it did from the editor setting.

Reply:
Same goes for it's use all over FSpatialGDKPackageAssembly